base/markdown/render/plain.jl

```diff
@@ -20,9 +20,12 @@ function plain{l}(io::IO, header::Header{l})
 end
 
 function plain(io::IO, code::Code)
-    println(io, "```", code.language)
+    # If the code includes a fenced block this will break parsing,
+    # so it must be enclosed by a longer ````-sequence.
+    n = mapreduce(length, max, matchall(r"`+", code.code)) + 1
```

Comment:
Ah yes, so it can be:

    mapreduce(length, max, 1, matchall(r"^`+"m, code.code))